### PR TITLE
Allows for TA to have faculty access

### PR DESCRIPTION
**Pull request by ndittren**
* Allows for TA to have faculty access

### CODE REVIEW

**Review by sdreher — COMMENTED** [review]
This looks like a good approach. Can you add a test?

File: lti_provider/mixins.py [hunk 1]
```diff
@@ -24,6 +24,12 @@ def join_groups(self, request, lti, ctx):
                 request.user.groups.add(ctx.faculty_group)
                 break
 
+            if settings.LTI_TOOL_CONFIGURATION['allow_ta_access'] is False:
```
Comment (sdreher) [inline lti_provider/mixins.py:27]: Your initial approach to this felt, e.g.

`if settings.LTI_TOOL_CONFIGURATION['allow_ta_access'] and 'teachingassistant' in role:`

What was your thinking around changing it?

File: lti_provider/tests/test_views.py [hunk 2]
```diff
@@ -50,6 +51,50 @@ def test_join_groups(self):
         self.assertTrue(user in ctx.group.user_set.all())
         self.assertTrue(user in ctx.faculty_group.user_set.all())
 
+    def test_join_groups_student(self):
```
Comment (sdreher) [inline lti_provider/tests/test_views.py:54]: Good catch in adding this test coverage.

File: lti_provider/mixins.py (hunk 1 above)
Comment (ndittren) [reply lti_provider/mixins.py]: No reason! While debugging I rearranged a lot. I missed changing this back.

**Pull request merged**